docs/user-guide/application_sources.md

```diff
@@ -1,4 +1,6 @@
-# Tools
+# Application Sources
```

Comment:
Can we keeps this as tools please?

Reply:
Sure. Although I think application sources is a bit more descriptive. "Tools" could mean argo-specific tools (among other things) not necessarily deployment tools.